pipelines/tekton/azureml-container-pipeline/kserve-download-model.yaml

```diff
@@ -12,13 +12,30 @@ spec:
     type: string
   - name: aws-bucket-name
     type: string
+  - name: fetch-github-model
+    type: string
+    default: "true"
+  - name: fetch-S3-model
+    type: string
+    default: "false"
+  - name: github-url
+    type: string
+    default: https://github.com/opendatahub-io/ai-edge/raw/3dd0242871371cf66af3f20e3a8924a4b5fc1658/pipelines/models/bike-rentals-auto-ml/model.pkl
```

Comment:
The image build process for the `bike-rentals-auto-ml` model needs more than just the `model.pkl` file, it includes the dependencies in `conda.yaml` for example. You should download the entire folder for the model.

Reply:
That's right, I did try moving it to the pipelineRun file, but it cause the pipelineRun creation to break. Now that you suggested I move to gitclone, I'll try moving it up again.